### ISSUE TITLE
required|array error

### ISSUE THREAD

**Issue by sculpepper** [poppinss/indicative#10]
When trying to validate a required array I am getting an issue. When I run the example provided in example.js cars also has the same failure. Hopefully the example makes sense.

Example:

``` javascript
//! This produces an error when validating saying that test is required even though it is present
var schema = {
  test : 'required|array'
};
data = {
  test : [0, 1]
}

//! Test is now being detected correctly and the error comes up saying it must be an array
data = {
  test: 1
}
```

**Comment by thetutlage** [comment]
Got it , due to nested schema , it is getting broken into sub rules. Working on same

For info

In indicative you can have nested object validations , so when you pass an array it is treating it another validation layer.

**Comment by sculpepper** [comment]
Thank you, exactly what I was looking for.

**Comment by thetutlage** [comment]
Fixed and backed it with over 15 tests :+1:  
Let me know if something else causes you issue , ideally it should not

published as v1.2.2
